src/main/java/seedu/duke/helper/Command.java

```diff
@@ -14,4 +14,25 @@ public void viewPatient(PatientList patientList, String parameters) {
         patientList.viewPatient(parameters);
     }
 
+    public void addPatient(PatientList patientList, String parameters) {
+        String[] parametersArray = Parser.parseAddPatient(parameters);
```

Comment:
Can we change the name to be plural since it is an array? Like arrayOfInputParameters or something?